packages/agent/src/agent/http/index.ts

```diff
@@ -386,8 +385,19 @@ export class HttpAgent implements Agent {
     });
 
     // Apply transform for identity.
-    transformedRequest = await id?.transformRequest(transformedRequest);
+    return id?.transformRequest(transformedRequest);
```

Comment:
@lmuntaner is removing this `await` intentional?

Reply:
No, just because I am returning the value, I don't think it's needed.

If we had a try/catch then yes, but since we don't, then I don't think so.